tests/src/whisk/core/cli/test/WskBasicUsageTests.scala

```diff
@@ -73,8 +73,9 @@ class WskBasicUsageTests
         Wsk.exists(wsk.usePythonCLI)
     }
 
-    it should "show help and usage info" in {
-        val stdout = wsk.cli(Seq("-h")).stdout
+    it should "show help and usage info using the default language" in {
+        val env = Map("LANG" -> "de_DE")
```

Comment:
That's what the test is for, no? When we have German strings, we'll have to update the test.